docs/distributed/DESIGN.md

```diff
@@ -103,6 +88,40 @@ Input for the key is the object name specified in `PutObject()`, returns a uniqu
 
 - MinIO does erasure coding at the object level not at the volume level, unlike other object storage vendors. This allows applications to choose different storage class by setting `x-amz-storage-class=STANDARD/REDUCED_REDUNDANCY` for each object uploads so effectively utilizing the capacity of the cluster. Additionally these can also be enforced using IAM policies to make sure the client uploads with correct HTTP headers.
 
+- MinIO also supports expansion of existing clusters in zones. Each zone is a self contained entity with same SLA's (read/write quorum) for each object as original cluster. By using the existing namespace for lookup validation MinIO ensures conflicting objects are not created. When no such object exists then MinIO simply uses the least used zone.
+
+*There are no limits on how many zones can be combined*
+
+```
+minio server http://host{1...32}/export{1...32} http://host{5...6}/export{1...8}
+```
+
+In above example there are two zones
+
+- 32 * 32 = 1024 drives zone1
+- 2 * 8 = 16 drives zone2
+
+> Notice the requirement of common SLA here original cluster had 1024 drives with 16 drives per erasure set, second zone is expected to have a minimum of 16 drives to match the original cluster SLA or it should be in multiples of 16.
```

Comment:
"second zone is expected to have a minimum of 16 drives"

I suggest it would be more accurate to say:
"second zone is required to have a multiple of 16 drives"

(for example: if you were to add 18 drives then that is more than a minimum of 16 drives; but it will not be accepted by minio as the zone will be configured as three sets of 6 drives)

Reply:
It does say that it's in multiples of 16

Reply:
It says: "to have a *minimum* of 16 drives to match the original cluster SLA *or* it should be in multiples of 16".  The implication of "a minimum of" is that any n>16 is OK, but that's not true.

Removing "a minimum of" fixes this.  Or you could say: "to have 16 drives (or a multiple of 16 drives) to match the original cluster SLA"

Even this is not always true.  To take a simple example: suppose I build a minio server initially with 4 drives.  I then add another zone with 8 drives (which is a multiple of 4).  However this will be rejected for inconsistent SLA, because the first zone is treated as 1x4 and the second zone as 1x8.

Reply:
I would suggest send a PR it's pointless to debate here.

Reply:
For all improvements send a PR, we are a free software project. Happy to accept relevant changes.